cmd/prometheus/main.go

```diff
@@ -61,14 +61,12 @@ import (
 
 var (
 	configSuccess = prometheus.NewGauge(prometheus.GaugeOpts{
-		Namespace: "prometheus",
-		Name:      "config_last_reload_successful",
-		Help:      "Whether the last configuration reload attempt was successful.",
+		Name: "prometheus_config_last_reload_successful",
```

Comment:
Note that this is equivalent to the old version. Not sure what the motivation behind the change here is. I leave it to @brian-brazil to decide if it makes sense to change it at this time.

Reply:
As you said it's equivalent in for metrics, but it will allow to search code by string and find match. I thinks it's also only case where namespace is used.

Reply:
I personally prefer having the full name for greppability.

Reply:
I don't feel strongly either way. I was just confused about the change here as it is unrelated to the purpose of this PR.
If @brian-brazil is fine with it, there is no need to take it out, I guess.